### PR TITLE
Move topology charts to enterprise, docs

### PR DESCRIPTION
**Pull request by jacobp100**

### CODE REVIEW

File: packages/ag-charts-community/src/chart/factory/expectedEnterpriseModules.ts [hunk 1]
```diff
@@ -39,16 +39,18 @@ export const EXPECTED_ENTERPRISE_MODULES: EnterpriseModuleStub[] = [
     { type: 'series', optionsKey: 'series[]', chartTypes: ['cartesian'], identifier: 'candlestick' },
     { type: 'series', optionsKey: 'series[]', chartTypes: ['cartesian'], identifier: 'bullet' },
     { type: 'series', optionsKey: 'series[]', chartTypes: ['cartesian'], identifier: 'heatmap' },
+    { type: 'series', optionsKey: 'series[]', chartTypes: ['cartesian'], identifier: 'range-area' },
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/factory/expectedEnterpriseModules.ts:42]: Good spot, thanks for this!

File: packages/ag-charts-community/src/options/series/topology/topologyOptions.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ export type AgTopologySeriesOptions = AgMapSeriesOptions | AgMapMarkerSeriesOpti
 export interface AgBaseTopologyChartOptions {
     /** Series configurations. */
     series?: AgTopologySeriesOptions[];
+    /** Topology to use in all series. */
+    topology?: any;
```
Comment (alantreadway) [inline packages/ag-charts-community/src/options/series/topology/topologyOptions.ts:11]: Nit: Could we use an alias type called `GeoJson` to give some hint as to what is expected here?

File: packages/ag-charts-enterprise/tsconfig.lib.json [hunk 3]
```diff
@@ -3,6 +3,10 @@
   "compilerOptions": {
     "types": ["node"]
   },
-  "include": ["src/**/*.ts"],
+  "include": [
+    "src/**/*.ts",
+    "../ag-charts-community/src/chart/series/topology/mercatorScale.ts",
+    "../ag-charts-community/src/chart/series/topology/lonLatBbox.ts"
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/tsconfig.lib.json]: This looks wrong, can you export these via the `main.ts` in `ag-charts-community` instead?

File: packages/ag-charts-enterprise/tsconfig.lib.json [hunk 4]
```diff
@@ -3,6 +3,10 @@
...
+  ],
```
Comment (jacobp100) [inline packages/ag-charts-enterprise/tsconfig.lib.json]:
```suggestion
  "include": ["src/**/*.ts"],
```

File: packages/ag-charts-community/src/options/series/topology/topologyOptions.ts (hunk 2 above)
Comment (jacobp100) [reply packages/ag-charts-community/src/options/series/topology/topologyOptions.ts:11]: Without a peer dep on `geojson`, it's not easily possible. I do want to add some runtime validation

**Comment by alantreadway** [comment]
(LGTM)

**Pull request merged**